contributors.json

```diff
@@ -102,6 +102,7 @@
   {"username":"mbifulco","contributions":[{"pr":372,"action":"typo fix"}]},
   {"username":"alessandroniciforo","contributions":[{"pr":366,"action":"fading parrot"}]},
   {"username":"itmart","contributions":[{"pr":358,"action":"tennis parrot"}]},
-  {"username":"gabrieltal","contributions":[{"pr":357,"action":"crying Jordan parrot"}]}
+  {"username":"gabrieltal","contributions":[{"pr":357,"action":"crying Jordan parrot"}]},
+  {"username":"adityaas26","contributions":[{"pr"406:,"action":"black bg parrot, ultra slow-mo parrot"}]}
```

Comment:
Small typo here, the `:` got on the wrong side of the number.  Thanks for updating this file though, it's a pain to go back and add them all later.

Reply:
Ohh god!
I'll fix it ASAP!
Thanks for your quick response!! 👍